test/producer-consumer-spec.js

```diff
@@ -141,4 +141,17 @@ describe('producer/consumer', function () {
       });
     });
   });
+
+  describe('undefined queue name', () => {
+    before(() => {
+      bunnymq.connection._config.consumerSuffix = undefined;
+    });
+
+    it('should receive message', () => {
+      bunnymq.consume('queue-name-undefined-suffix', (message) => {
+        assert.equal(message.msg, 'test for undefined queue suffix');
+      })
```

Comment:
Hm what are the pros/cons ?
Isn't it kinda the same ? If it improves the quality overall i'd be glad to change !